docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:11.6
+
+ARG JOBS=1
+
+ENV TZ America/Sao_Paulo
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt update && \
```

Comment:
Never got a confirmation dialog when `apt update`ing. Skimming through the [`update` source](https://github.com/Debian/apt/blob/aa8c9aad81b57da96e8ac7345d0024ee526c7714/apt-private/private-update.cc) (main branch), I didn't see any usage of the `APT::Get::Assume-Yes` option nor any `YnPrompt` (used for displaying the "Do you want to continue?" interactive message).

But we may set it just in case I missed something.